### DISCUSSION TITLE
Session Undefined Error

### DISCUSSION THREAD

**Discussion by CharlieAtki** [community/community#147108] [Programming Help]
### Body

Hi, I have recently developed two modules of an Enterprise Resource Planner (ERP) for my OCR A-Level Computer Science NEA, but during my modifications to allow for web hosting, my generateTwoFactor API endpoint was unable to access the created sessions, which was generated on user login. 

I have now spent a large amount of time attempting to debug the code myself, but I am unable to identify the root cause of the session inaccessibility even though the session is being created and stored within mongoDB.

Info:
- MERN stack 
- Tailwind CSS
- Recharts

Below is the URL to my projects repository. Please comment for any extra information about my issue. 

[https://github.com/CharlieAtki/ERP](https://github.com/CharlieAtki/ERP)

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by OtayNacef** [comment]
It seems like you're hosting the frontend and attempting to call a local backend,
I suggest hosting your backend server as well or using tools like ngrok to expose your local backend temporarily for testing with the frontend
